server/src/app.controller.ts

```diff
@@ -7,20 +7,27 @@ import {
 } from '@nestjs/common';
 import { Response } from 'express';
 import { AuthService } from './auth/auth.service';
+import { ContactService } from './contact/contact.service';
 
 @Controller()
 export class AppController {
   constructor(
     private readonly authService: AuthService,
+    private readonly contactService: ContactService,
   ) {}
 
   @Get('/login')
   async login(@Res() res: Response, @Query('accessToken') NYCIDToken: string) {
     try {
       const ZAPToken = await this.authService.generateNewToken(NYCIDToken);
+      const { contactId } = await this.authService.validateCurrentToken(ZAPToken);
+      const { contactid, emailaddress1 } = await this.contactService.findOneById(contactId);
 
-      res.cookie('token', ZAPToken, { httpOnly: true, sameSite: 'none', secure: true })
-        .send({ message: 'Login successful!' });
+      res.send({
+        access_token: ZAPToken,
+        contactid,
```

Comment:
Interestingly enough, I don't think the frontend actually needs the contactid 😂  It's not used in the frontend, and the backend looks up the contactid each time using the auth token in the request header.

Reply:
Well, I guess the contactid is already embedded within the token

Reply:
Oooh. Good catch. I'll clean that up. IDK why I included it anyway.